### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by fxw8934** [getlantern/lantern-forum-cn#1095] [General]
开通了专业版还没免费版好用，虽然免费版也不好用。最幽默的是我访问这个论坛还需要用其他VPN，蓝灯自己的VPN都登不上来。
![微信图片_20241221000813](https://github.com/user-attachments/assets/c044b2c0-f3ec-4e83-8ff9-7eb4aa065fcf)
刚开通不到一个小时就准备退款了，半天访问不了论坛，买了个其他VPN才登上来的

**Comment by fxw8934** [comment]
账号 1711872975@qq.com

**Comment by MaxMKMax** [comment]
@fxw8934 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

在帮您退款之前. 需要您帮个忙, 我们才能帮您退款! 
我们也想知道为什么Lantern无法登入GitHub...

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by fxw8934** [reply]
80497302，请尽快退款，谢谢

**Comment by fxw8934** [comment]
大概多久能退款成功

**Comment by fxw8934** [comment]
@Mike777Khoo 请问什么时候能处理好退款

**Comment by MaxMKMax** [comment]
@fxw8934 不好意思让您久等了. 我帮您问一下.
